source/JumpTypes.h

```diff
@@ -0,0 +1,27 @@
+/* JumpTypes.h
+Copyright (c) 2022 by Amazinite
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef JUMP_TYPES_H_
+#define JUMP_TYPES_H_
+
+enum class JumpType : int {
```

Comment:
short comment here maybe:
```suggestion
// All jump types that can be done. Return type for navigation.
enum class JumpType : int {
```

Not a very good comment, but something that explains this file.